### DISCUSSION TITLE
Alienware m16 R1 - Fan Control - help needed

### DISCUSSION THREAD

**Discussion by Antergosgeek** [T-Troll/alienfx-tools#424] [Q&A]
Alienware m16 R1 here is what it looks like:

<img width="233" alt="image" src="https://github.com/T-Troll/alienfx-tools/assets/26154541/4c418e3e-5f75-4187-badf-107b265585e7">

At this point I hope I am using the right sensors. Am a little confused to be honest.

Can anyone help?

Also does anyone have a suggested fan curve that works well?

**Comment by Wromper** [comment]
I have the m16 R1 AMD and having a lot of issues with fan noise.  Even on 'quiet' and 'balanced' presets, the fans can spin over 3000 RPM at low temperatures like 70C.  I tried the AlienFX-Fan tool, set the curve flat on all 4 fans, and it still doesnt prevent the fans from spinning wildly.  I even saw all 4 fans at 100% with the 4 internal temps at 75C, 70C, 75C and 69C.  HELP!  I know the Alien-Fan tool is doing something because changing the power mode to 'disabled' turns off the CPU boost clock.  All the other modes seem to do exactly the same thing though, allow boost and CPU power at 60 watts.  thanks, help.

**Reply by T-Troll** [reply]
As i told you, it's a BAD idea to slow down fans at high 70s.
At normal operation (low load) CPU should be between 40 and 50.
PASV (stop fans) point at 46C for m16, AFAIR.
